tardis/montecarlo/montecarlo_numba/base.py

```diff
@@ -220,7 +88,9 @@ def montecarlo_main_loop(
     n_threads = get_num_threads()
 
     estimator_list = List()
-    for i in range(n_threads):  # betting get tid goes from 0 to num threads
+    for i in range(
+        n_threads
```

Comment:
Why does this get kicked a line down? Seems excessive.

Reply:
That's what black wants AFAICT

Reply:
done